logs_api/lib/opentelemetry/logs/logger_provider.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Logs
+    # No-op implementation of a logger provider
+    class LoggerProvider
+      # Returns a {Logger} instance.
+      #
+      # @param name [optional String] Instrumentation package name
+      # @param version [optional String] Instrumentation package version
+      # @param schema_url [optional String] Schema URL recorded in the emitted
+      #   telemetry.
+      # @param attributes [optional Hash] Specifies the instrumentation scope
+      #   attributes to associate with emitted telemetry.
+      #
+      # @return [OpenTelemetry::Logs::Logger]
+      #
+      # @api public
+      def logger(name: nil, version: nil, schema_url: nil, attributes: {})
```

Comment:
The methods I've added are using named arguments. It looks like the SDK and API use them more sparingly. Is there a preference for one approach over the other?